ScipDotnet/ScipIndexer.cs

```diff
@@ -13,6 +13,8 @@ namespace ScipDotnet;
 /// </summary>
 public class ScipIndexer
 {
+    private const int DotnetRestoreTimeout = 3000;
```

Comment:
On a slow connection, is it normal for resolve to take longer than 3 seconds? I'd be fine with setting this timeout to a very high value like 5 minutes, and even making it configurable. We can do this in a separate PR, it's fantastic to merge this now to get a stable CI.

Reply:
I liked your suggestion, so I have made the following PR: https://github.com/sourcegraph/scip-dotnet/pull/31